### DISCUSSION TITLE
Is there a way to disable all animations and smooth scrolling?

### DISCUSSION THREAD

**Discussion by vctls** [Zettlr/Zettlr#4062] [Q&A] [answered]
Is there some way of disabling all animations, transitions, fades, etc?
And also smooth scrolling?

Maybe through Electron flags?

It would be nice if it took the global "prefer reduced motion" setting of the OS.

I didn't find anything in the docs, issues or discussions.

**Comment by vctls** [comment]
Oh, I found something for the transitions already! 😅
Just put this into the `custom.css`:
```css
* {
  transition: none !important;
}
```

For smooth scrolling, I think Electon should be able to take a ` --disable-smooth-scrolling` option, but I'm not sure where to put that with a Flatpak install…

**Reply by vctls** [reply, marked as answer]
Huh, this works, actually:
```bash
/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=start-zettlr com.zettlr.Zettlr --disable-smooth-scrolling
```
It can be added in a `.desktop` file.

**Comment by exanrauzer** [comment]
When you scroll over text area, you got smooth scrolling, but when you scroll over left margin the smooth scrolling is disabled.

**Reply by vctls** [reply]
Wow… thanks!
Did I miss that in the docs?

**Reply by exanrauzer** [reply]
Its not a bug - Its a feature!😁

**Reply by vctls** [reply]
Well, it seems v3 fixed the "feature" :cry:
